src/broker/cas_execute.c

```diff
@@ -3643,7 +3643,8 @@ get_column_default_as_string (DB_ATTRIBUTE * attr, bool * alloc)
 {
   DB_VALUE *def = NULL;
   int err;
-  char *default_value_string = NULL;
+  char *default_value_string = NULL, *default_expr_format = NULL;
```

Comment:
I think it would be better to refactor the function to have 3 parts:

```c
  def = db_attribute_default (attr);
  if (def == NULL)
    {
      /* no default clause case */
      return default_value_string;
    }

  if ( ... )
    {
      /* default expression case */
    }
  else
    {
      /* default value case */
    }
```